### ISSUE TITLE
SymbolDictionary browser completely broken by an unreadable value in a Global

### ISSUE THREAD

**Issue by LisaAlmarode** [ericwinger/Jade#582]
if you do something like this in topaz:
run
UserGlobals at: #bad put: '£¥š' asSymbol
%
so that:
run
bad class name
%
DoubleByteSymbol

You can login but any attempt to go to the (NONE) dictionary UserGlobals will result in 
an error Can't gold $\x161.  You cannot look at anything in this dictionary at all.

**Comment by ericwinger** [comment]
@LisaAlmarode - I couldn't reproduce the exact problem but I did find a similar problem which I fixed. Can you put this code into ``RowanDictionaryService`` replacing the existing method and see if it fixes the specific case you're seeing? 

```
update
  | dictionary sorted |
  classes := Array new.
  sorted := SortedCollection sortBlock: [ :x :y | x first < y first ].
  dictionary := Rowan image symbolList objectNamed: name.
  dictionary ifNil: [ ^ self ].
  dictionary
    keysAndValuesDo: [ :key :value | 
      value isClass
        ifTrue: [ 
          | classService |
          classService := RowanClassService new name: key asString.
          classService versions: value classHistory size.
          classService version: (value classHistory indexOf: value).
          classes add: classService ]
        ifFalse: [ 
          | printString theKey |
          printString := [ value printString ]
            on: Error
            do: [ :ex | 'unprintable string. Error - <' , ex printString , '>' ].
          key charSize = 1
            ifTrue: [ theKey := key ]
            ifFalse: [ theKey := '<<unprintable string. charSize > 1>>' ].
          sorted
            add:
              (Array
                with: name , '.' , theKey
                with: value class name
                with: value asOop
                with: printString) ] ].
  globals := sorted asArray.
  RowanCommandResult addResult: self
```

**Comment by ericwinger** [comment]
No feedback yet. Believed fixed in 3.0.82.

**Comment by LisaAlmarode** [comment]
In 3.0.82, it's handling this okay.  Globals even points out charsize > 1, and the inspector doesn't do badly either.
